Makefile

```diff
@@ -387,14 +387,11 @@ cloud-build-prerequisites:
 	apk add --no-cache jq
 
 .PHONY: release-staging
-release-staging: ## Release the cloud provider images.
-ifeq ($(CLOUD_BUILD_IMAGE),ccm)
-	ENABLE_GIT_COMMAND=$(ENABLE_GIT_COMMAND) $(MAKE) build-all-ccm-images push-multi-arch-controller-manager-image
-else
-	ENABLE_GIT_COMMAND=$(ENABLE_GIT_COMMAND) $(MAKE) cloud-build-prerequisites build-all-node-images push-multi-arch-node-manager-image
-endif
+release-staging:
+	ENABLE_GIT_COMMANDS=false IMAGE_REGISTRY=$(STAGING_REGISTRY) $(MAKE) build-images push-images
+
```

Comment:
Do we use this, wondering if we need this change?

Reply:
Looks like it came in from upstream as I was cherry-picking our stuff onto the Makefile.

https://github.com/kubernetes-sigs/cloud-provider-azure/blob/master/Makefile#L390 is where it's defined and https://github.com/kubernetes-sigs/cloud-provider-azure/blob/master/cloudbuild.yaml#L22 is where it's used.

I'd say we should keep it to make future rebases easier, but I'm not overly attached to that idea, either.

Reply:
Yeah my point was more, I don't understand why our commit changes this part of the makefile, should it not just be adding the include?

Reply:
I think I see what I did; will fix.

Reply:
Fixed.